### PR TITLE
chore(docs): Microsoft Entra ID typos

### PR DESCRIPTION
**Pull request by mariusch**
## ☕️ Reasoning

Ran into some minor issues when upgrading to version 5. The name of the provider should be MicrosoftEntraID. NextAuth exports signIn and signOut in camelCase. 

## 🧢 Checklist

- [x] Documentation
- [x] Tests
- [x] Ready to be merged

## 📌 Resources

- [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md)
- [Contributing guidelines](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md)
- [Code of conduct](https://github.com/nextauthjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

**Review by kongebra — APPROVED** [review]
Agree on the import naming here. `MicrosoftEntraID` is better for the documentations. 😊

**Comment by ndom91** [comment]
Hey thanks for the camelCase signIn related changes. 

Regarding the Entra import, that provider is a default export so you should be able to name it whatever you want. 

Can you double check that leaving it as `Entra` really doesn't work for you? 🤔

**Review by ndom91 — APPROVED** [review]
Can confirm, importing as `Entra` does still work as expected. However, `MicrosoftEntraId` is technically more correct so I'll take this PR as-is. 

Thanks! :pray:

File: docs/pages/getting-started/providers/microsoft-entra-id.mdx [hunk 1]
```diff
@@ -57,9 +57,9 @@ AUTH_MICROSOFT_ENTRA_ID_TENANT_ID
 
 ```ts filename="/auth.ts"
 import NextAuth from "next-auth"
-import Entra from "next-auth/providers/microsoft-entra-id"
+import MicrosoftEntraID from "next-auth/providers/microsoft-entra-id";
```
Comment (ndom91) [inline docs/pages/getting-started/providers/microsoft-entra-id.mdx:60]: Breaks linting

```suggestion
import MicrosoftEntraID from "next-auth/providers/microsoft-entra-id"
```

**Pull request merged**